### ISSUE TITLE
Add the ability to redefine the entity identifier

### ISSUE THREAD

**Issue by xtrn143** [sedyh/mizu#7]
i want every entity has a id which i custom id,then i can use w.GetEntity(id) to obtain the entity which i want update.

**Comment by sedyh** [comment]
Hello, did I understand correctly that you need the ability to set or redefine your own identifier for entities?

**Comment by xtrn143** [comment]
yes,because i need update the specified entity’s component data from backend

**Comment by sedyh** [comment]
Okay, I'll try to do something about it. A separate method or verification when getting the ID should be suitable.

**Comment by xtrn143** [comment]
Thanks
